web/db/actions/Post.ts

```diff
@@ -131,6 +187,15 @@ async function getAllPosts() {
   return await Post.find().sort({ date: -1 });
 }
 
+async function getAttachments(oid: ObjectId) {
+  const listObjectsCommand = new ListObjectsCommand({
+    Bucket: consts.storageBucket,
+    Prefix: oid + "",
```

Comment:
What's the purpose of `+ ""`? Is it to turn `oid` into a string? If so, you can try using type assertion to assert `oid as string`

Reply:
Yes, it was. It was because `oid as string` threw an error. I've changed it to `oid as unknown as string` now.

Reply:
Alright, gotcha. Could you try `oid.toString()`?